python/notebooks/06-memory-and-embeddings.ipynb

```diff
@@ -11,10 +11,10 @@
     "So far, we've mostly been treating the kernel as a stateless orchestration engine.\n",
     "We send text into a model API and receive text out.\n",
     "\n",
-    "In a [previous notebook](04-context-variables-chat.ipynb), we used `context variables` to pass in additional\n",
-    "text into prompts to enrich them with more context. This allowed us to create a basic chat experience.\n",
+    "In a [previous notebook](04-kernel-arguments-chat.ipynb), we used `kernel arguments` to pass in additional\n",
+    "text into prompts to enrich them with more data. This allowed us to create a basic chat experience.\n",
     "\n",
-    "However, if you solely relied on context variables, you would quickly realize that eventually your prompt\n",
+    "However, if you solely relied on kernel arguments, you would quickly realize that eventually your prompt\n",
```

Comment:
This change in this line is not a fix link, but a change in wording. It was changed to match the [06-memory-and-embedding in dotnet](https://github.com/microsoft/semantic-kernel/blob/14a0881989e2bd23c75568fe196c705f9bb65f79/dotnet/notebooks/06-memory-and-embeddings.ipynb#L16).

Reply:
Sorry, and thanks